docs/howto/error-reporting.txt

```diff
@@ -123,7 +123,7 @@ Filtering error reports
     Filtering sensitive data is a hard problem, and it's nearly impossible to
     guarantee that sensitive data won't leak into an error report. Therefore,
     error reports should only be available to trusted team members and you
-    should avoid transmitting error reports unencrypted over the Internet
+    should avoid transmitting error reports unencrypted over the internet
```

Comment:
@felixxm -- It's late here so maybe it's quite possible I'm missing something, but is this the change you mention?

(It's difficult to see the Internet through all the the Internet Explorers)

Reply:
Yes, this line.